### ISSUE TITLE
warnings.warn("Error parsing BridgeSupport data

### ISSUE THREAD

**Issue by tjluoma** [univ-of-utah-marriott-library-apple/display_manager#4]
Running `display_manager.py show` on my Mac (running 10.14 (18A365a)) gives this warning:

/System/Library/Frameworks/Python.framework/Versions/2.7/Extras/lib/python/PyObjC/objc/_bridgesupport.py:674: RuntimeWarning: Error parsing BridgeSupport data for IOKit: PyObjCRT_SkipTypeSpec: Unhandled type '0'
  warnings.warn("Error parsing BridgeSupport data for %s: %s" % (frameworkName, e), RuntimeWarning)

before it outputs the expected information:

```
display "main":
resolution:     1440x900
refresh rate:   0
HiDPI:          True
rotation:       0
brightness:     1.00
```

Similarly `display_manager.py help` gives this warning:

/System/Library/Frameworks/Python.framework/Versions/2.7/Extras/lib/python/PyObjC/objc/_bridgesupport.py:674: RuntimeWarning: Error parsing BridgeSupport data for IOKit: PyObjCRT_SkipTypeSpec: Unhandled type '0'
  warnings.warn("Error parsing BridgeSupport data for %s: %s" % (frameworkName, e), RuntimeWarning)

before showing the expected

```
usage:  display_manager.py <command>

COMMANDS (required)
    help        Show help information about a command
    show        Show current/available display configurations
    res         Manage display resolution
    brightness  Manage display brightness
    rotate      Manage display rotation
    underscan   Manage display underscan
    mirror      Manage screen mirroring
```

**Comment by uurazzle** [comment]
Hello @tjluoma:

Thanks for taking the time to post back about this issue. We will investigate and troubleshoot and reply back on status.

**Comment by ahdavies6** [comment]
@tjluoma,

Thank you for letting us know about this issue. We've been able to reproduce it, and are investigating the cause right now. However, it looks like attempting to set the resolution, brightness, rotation, underscan, or mirroring all work as intended (albeit with the unexpected `RuntimeWarning`). Let us know if you've experienced a different result, and we'll try to get to the root of the problem soon.

**Comment by tjluoma** [comment]
The error is distracting, but everything seems to work fine.

**Comment by ahdavies6** [comment]
@tjluoma 

Thanks again for letting us know about the issue. We've found the source, and it won't affect Display Manager -- PyObjC is simply sending a general warning about incompatibility within a part of IOKit that we don't use. Everything else is working, so we've suppressed the warning in [v1.0.1](https://github.com/univ-of-utah-marriott-library-apple/display_manager/tree/v1.0.1). If you'd like to reinstall using the latest installer, you'll no longer be distracted by the warning.
